src/segment_node_test.go

```diff
@@ -25,9 +36,7 @@ func TestNodeMatchesVersionFile(t *testing.T) {
 			language: &language{
 				env: env,
 				activeCommand: &cmd{
-					version: &version{
-						Full: tc.Version,
-					},
+					version: &tc.Version,
```

Comment:
if it's always the same, why not add the above version here directly?

Reply:
Yes, of course...